name-as/src/main.rs

```diff
@@ -25,7 +25,25 @@ fn main() -> Result<(), &'static str> {
     } else {
         // Otherwise, use provided assembler command
         println!("Config Name:   {}", config.config_name);
-        println!("Assembler CMD: {}", config.as_cmd);
+        println!("Assembler CMD: {:?}", config.as_cmd);
+
+        for cmd in &config.as_cmd {
+            match Command::new("sh").arg("-c").arg(cmd.as_str()).output() {
```

Comment:
Not cross-platform. Remember what Cole said— 90% of CS majors use Windows :(

Reply:
Resolved (I think, need someone to make a Windows config and test it)